files/content/information_kiosk/information_hamis.lua

```diff
@@ -73,9 +78,63 @@ local tips = {
 	"I'll get so\nI'll get so\nI'll get so\nI'll get so\nI'll get so\nI'll get so\nI'll get so\nI'll get so\nI'll get so\nI'll get so\nI'll get so\nI'll get so\nI'll get so\nI'll get so",
 	"Death comes for those who wait.",
 	"what",
+	"Hello, " .. uid or useed or "{@color FF0000}ERROR",
+	"I know you!",
+	"Sorry, who are you?",
+	"Jump off a bridge!",
+	"{@color FF0000}#KYS#!!!!!!!!!!{@pause 90}\n{@color d991de}(Keep Yourself Safe <3 )",
+	"I'll get by, ~one {@color f0c854}gold{@color FFFFFF} at a time!\\~~",
+	"The Voices, they speak through me!",
+	'psst, try this secret cheatcode: "photocopier"',
+	"But before you get your tip, I would like to take a minute\nto thank today's Sponsor:{@pause 60}\n{@delay 30}...{@pause 60}{@delay 3}\nWe...{@pause 15} don't have any sponsors...",
+	"The factory hämmies will eat well tonight!",
+}
+
+if uid then table.insert(tips, "Higher beings, these words are for you alone.") end
+
+local seasonal = {
```

Comment:
I have not implemented seasonal or streamer tips, ill work on that soon i just wanted to push everything i had and take a break